src/processors/modulation/Flanger.cpp

```diff
@@ -0,0 +1,298 @@
+#include "Flanger.h"
+#include "../BufferHelpers.h"
+#include "../ParameterHelpers.h"
+
+namespace
+{
+constexpr float rateLow = 0.5f;
+constexpr float rateHigh = 40.0f;
+constexpr float delayMs = 0.001f;
+
+const String delayTypeTag = "delay_type";
+} // namespace
+
+Flanger::Flanger (UndoManager* um) : BaseProcessor ("Flanger",
+                                                    createParameterLayout(),
+                                                    um,
+                                                    magic_enum::enum_count<InputPort>(),
+                                                    magic_enum::enum_count<OutputPort>())
+{
+    using namespace ParameterHelpers;
+    loadParameterPointer (rateParam, vts, "rate");
+    loadParameterPointer (delayAmountParam, vts, "delayAmount");
+    loadParameterPointer (delayOffsetParam, vts, "delayOffset");
+    loadParameterPointer (fbParam, vts, "feedback");
+    loadParameterPointer (mixParam, vts, "mix");
+    delayTypeParam = vts.getRawParameterValue (delayTypeTag);
+
+    addPopupMenuParameter (delayTypeTag);
+
+    uiOptions.backgroundColour = uiOptions.backgroundColour = Colour (48, 25, 188);
```

Comment:
Blue colour I was experimenting with, definitely some other colour options to go with.

Reply:
I was thinking about the Boss pedals and how the chorus is blue and the flanger is purple. In our case the chorus is already purple so maybe some other shade of blue? I can do some experimenting with some other blue/purple colours. Is there any colours you had in mind?